.github/workflows/build_main.yml

```diff
@@ -156,9 +173,32 @@ jobs:
       matrix:
         include:
           - os: windows-2019
-            framework: 'net5'
+            architecture: 'AnyCPU'
+            framework: 'net48'
           - os: windows-2019
+            architecture: 'x86'
             framework: 'net48'
+          - os: windows-2019
+            architecture: 'x64'
+            framework: 'net48'
+          - os: windows-2019
+            architecture: 'AnyCPU'
+            framework: 'net5'
+          - os: windows-2019
+            architecture: 'x86'
+            framework: 'net5'
+          - os: windows-2019
+            architecture: 'x64'
+            framework: 'net5'
+          - os: windows-2019
+            architecture: 'AnyCPU'
+            framework: 'net6'
+          - os: windows-2019
+            architecture: 'x86'
+            framework: 'net6'
+          - os: windows-2019
+            architecture: 'x64'
+            framework: 'net6'
```

Comment:
The winforms build is out of sync with what is happening in the PR build. Can you take a look and make sure the `build_main.yml` is correct. There should only be 1 build command like this:
```
- name: Build WinForms App
  run: dotnet build -c Release /p:Platform=${{ matrix.architecture }}
  working-directory: ./samples/WinformsApp/WinformsApp.${{ matrix.framework }}
```